velox/common/io/IoStatistics.h

```diff
@@ -73,6 +73,13 @@ class IoCounter {
     casLoop(max_, other.max_, std::less());
   }
 
+  void clear() {
```

Comment:
Just do `ioCounter = {}` should be enough, there is no need to add an extra method for this